tezos/protocols.go

```diff
@@ -111,7 +111,7 @@ var (
 			{PtMumbai, 0, 2162689, 2957312, 528, 3, 8192, 512},  // v16
 			{PtNairobi, 0, 2957313, 5316608, 625, 3, 8192, 512}, // v17
 			{Proxford, 0, 5316609, 6422528, 913, 3, 8192, 512},  // v18
-			{PtParisB, 0, 6422529, -1, 913, 3, 8192, 8192},      // v19
+			{PtParisB, 0, 6422529, -1, 1048, 3, 12288, 0},       // v19
```

Comment:
BlocksPerSnapshot cannot be 0. This will result in divide by zero error in params.go